doc/src/learn/architecture/consensus.md

```diff
@@ -1,20 +1,23 @@
-- agreeing on a specific ordering of this data ([Bullshark](https://arxiv.org/abs/2201.05677))
+- ensuring the availability of data submitted to consensus = [Narwhal](https://arxiv.org/abs/2105.11827)
+- agreeing on a specific ordering of this data = [Bullshark](https://arxiv.org/abs/2201.05677) or [Tusk ]
 
-> **Note:** Bullshark has replaced the Tusk component of the consensus protocol for reduced latency and support for fairness (where even slow validators can contribute). See [DAG Meets BFT - The Next Generation of BFT Consensus](https://decentralizedthoughts.github.io/2022-06-28-DAG-meets-BFT/) for a comparison of the protocols.
+In August 2022, Bullshark replaced the Tusk component of the consensus protocol as the default for reduced latency and support for fairness (where even slow validators can contribute). See [DAG Meets BFT - The Next Generation of BFT Consensus](https://decentralizedthoughts.github.io/2022-06-28-DAG-meets-BFT/) for a comparison of the protocols.
 
-This is done in two layered modules, so Narwhal can also be used coupled with an external consensus algorithm, such as HotStuff, Istanbul BFT, or Tendermint. Narwhal is undergoing integration in the [Celo](https://www.youtube.com/watch?v=Lwheo3jhAZM) and [Sommelier](https://www.prnewswire.com/news-releases/sommelier-partners-with-mysten-labs-to-make-the-cosmos-blockchain-the-fastest-on-the-planet-301381122.html) blockchain.
+Still, you may easily use Tusk instead of Bullshark by reverting the change shown in:
+https://github.com/MystenLabs/narwhal/blob/85c226f2824010ff695d0bc5789a24cad2bce289/node/src/lib.rs#L266
 
-Narwhal and Bullshark represent the latest variant of decades of work on multi-proposer, high-throughput consensus algorithms that reaches throughputs more than 125,000 transactions per second with a two-second latency for a deployment of 50 parties, with production cryptography, permanent storage, and a scaled-out primary-worker architecture.
+Consensus is accomplished in two layered modules, so Narwhal can also be used coupled with an external consensus algorithm, such as HotStuff, Istanbul BFT, or Tendermint. Narwhal is undergoing integration in the [Celo](https://www.youtube.com/watch?v=Lwheo3jhAZM) and [Sommelier](https://www.prnewswire.com/news-releases/sommelier-partners-with-mysten-labs-to-make-the-cosmos-blockchain-the-fastest-on-the-planet-301381122.html) blockchain.
 
-The Narwhal/Bullshark approach can offer dramatic scalability benefits in the following cases:
+The Sui Consensus Engine represents the latest variant of decades of work on multi-proposer, high-throughput consensus algorithms that reaches throughputs more than 125,000 transactions per second with a two-second latency for a deployment of 50 parties, with production cryptography, permanent storage, and a scaled-out primary-worker architecture.
+
+The Sui Consensus Engine approach can offer dramatic scalability benefits in the following cases:
```

Comment:
approach can offer or does offer?
approach offers dramatic

Reply:
I think we should hedge and not say it always does. @huitseeker , what do you think?

Reply:
Yes, the original intent of this sentence was to hedge: there are reasonable but unmentioned requirements to reaping all the benefits, and the reader's use case may fall outside those requirements.

Reply:
Thanks, @huitseeker